CMakeLists.txt

```diff
@@ -313,6 +313,8 @@ if(AVIF_BUILD_TESTS)
     )
 endif()
 
+configure_file(libavif.pc.cmake ${CMAKE_CURRENT_BINARY_DIR}/libavif.pc)
```

Comment:
I would recommend to use `@ONLY` and make the PC file using the variables in the `@PROJECT_NAME@` style to make sure we can use `${var}` in the shell script sense

Reply:
Done, thanks.